packages/eslint-plugin/docs/rules/no-base-to-string.mdx

```diff
@@ -64,6 +66,15 @@ const literalWithToString = {
 </TabItem>
 </Tabs>
 
+## Alternatives
+
+Consider using `JSON.stringify` when you want to convert non-primitive things to string for logging, debugging, etc.
+
+```typescript
+declare const o: object;
+const errorMessage = 'Found unexpected value: ' + JSON.stringify(o);
+```
```

Comment:
Hi @kirkwaiblinger, We apologize for the delay in responding. I was on vacation for a while. I added documentation for alternative methods. thanks!